### PR TITLE
Undoes global food nutriment nerf

### PR DESCRIPTION
**Pull request by Heroman3003**
Title. Undoes global nerf that halved amount of nutrition in all food introduced as part of #10716

### CODE REVIEW

File: code/modules/food/food/snacks.dm [hunk 1]
```diff
@@ -27,7 +27,7 @@
 /obj/item/weapon/reagent_containers/food/snacks/Initialize()
 	. = ..()
 	if(nutriment_amt)
-		reagents.add_reagent("nutriment",nutriment_amt,nutriment_desc)
+		reagents.add_reagent("nutriment",(nutriment_amt*2),nutriment_desc)
```
Comment (Very-Soft) [inline code/modules/food/food/snacks.dm:30]: Should this not be marked with vorestation edit?

Comment (Heroman3003) [reply code/modules/food/food/snacks.dm:30]: Oh whoopsie daisy mamma mia

**Comment by Ketrai** [comment]
Whilst I'm down for food being very filling. There's a lot of food items that are actually too filling currently because of this error. It's extremely confusing for anyone new trying to implement a food item. And it should definitely be fixed. Maybe just multiply all nutriment values by 2 for vore foods for now? Not that many items in that list.

**Comment by Heroman3003** [comment]
> 
> 
> Whilst I'm down for food being very filling. There's a lot of food items that are actually too filling currently because of this error. It's extremely confusing for anyone new trying to implement a food item. And it should definitely be fixed. Maybe just manually multiply all nutriment values by 2 for vore foods for now? Not that many items in that list.

Its not even mainly about vore foods as it is about rations and vending snacks. If someone wants to manually remove the multiplier and then manually go through every food item editing it, they are welcome to go ahead, I'm just undoing the less-than-desired netf to food as a whole.

**Comment by Arokha** [comment]
Was discussed in staff meeting and approved

**Pull request merged**